docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -18,77 +17,56 @@
+If the out-of-the-box conversion (only the ``input_model`` parameter is specified) is not successful, use the parameters mentioned below to override input shapes and cut the model:
 
-- Model Optimizer provides two parameters to override original input shapes for model conversion: ``--input`` and ``--input_shape``.
+- model conversion API provides two parameters to override original input shapes for model conversion: ``input`` and ``input_shape``.
 For more information about these parameters, refer to the :doc:`Setting Input Shapes <openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model>` guide.
 
 - To cut off unwanted parts of a model (such as unsupported operations and training sub-graphs),
-use the ``--input`` and ``--output`` parameters to define new inputs and outputs of the converted model.
+use the ``input`` and ``output`` parameters to define new inputs and outputs of the converted model.
 For a more detailed description, refer to the :doc:`Cutting Off Parts of a Model <openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model>` guide.
 
 You can also insert additional input pre-processing sub-graphs into the converted model by using
-the ``--mean_values``, ``scales_values``, ``--layout``, and other parameters described
+the ``mean_values``, ``scales_values``, ``layout``, and other parameters described
 in the :doc:`Embedding Preprocessing Computation <openvino_docs_MO_DG_Additional_Optimization_Use_Cases>` article.
 
-The ``--compress_to_fp16`` compression parameter in Model Optimizer allows generating IR with constants (for example, weights for convolutions and matrix multiplications) compressed to ``FP16`` data type. For more details, refer to the :doc:`Compression of a Model to FP16 <openvino_docs_MO_DG_FP16_Compression>` guide.
+The ``compress_to_fp16`` compression parameter in model conversion API allows generating IR with constants (for example, weights for convolutions and matrix multiplications) compressed to ``FP16`` data type. For more details, refer to the :doc:`Compression of a Model to FP16 <openvino_docs_MO_DG_FP16_Compression>` guide.
 
-To get the full list of conversion parameters available in Model Optimizer, run the following command:
+To get the full list of conversion parameters available in model conversion API, run the following command:
 
 .. code-block:: sh
 
    mo --help
 
 
-Examples of CLI Commands
-########################
+Examples of Conversion API parameters
+#####################################
 
-Below is a list of separate examples for different frameworks and Model Optimizer parameters:
+Below is a list of separate examples for different frameworks and model conversion API parameters:
 
-1. Launch Model Optimizer for a TensorFlow MobileNet model in the binary protobuf format:
+1. Launch model conversion API for a TensorFlow MobileNet model in the binary protobuf format:
```

Comment:
Below, the command-line is for the tool and not for Model Conversion API. So here we have misalignment between what you say and command shared below